app/app.go

```diff
@@ -210,7 +210,7 @@ func NewStakebirdApp(
 ) *StakebirdApp {
 
 	// TODO: Remove cdc in favor of appCodec once all modules are migrated.
-	cdc := std.MakeCodec(ModuleBasics)
+	cdc := std.MakeCodec(ModuleBasics) // nolint
```

Comment:
Adding `nolint` here as there is a TODO on the line above.

Reply:
I think this line can be removed